docs/connections.md

```diff
@@ -129,7 +129,8 @@ via `pip install teradatasql` if you have a license.
     # Connection Details
     "host": "127.0.0.1",
     "port":1025,
-    "user_name":"my-user",
+    "logmech":"TD2",
+    "user-name":"my-user",
```

Comment:
This won't work since in the connection config JSON expects an underscore. For adding a connection via CLI, we expect a hyphen and the [code in cli_tools](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/cli_tools.py#L338) manages that. Could you revert this change?

Instead, there is a bug in the connections documentation [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/docs/connections.md?plain=1#L30) that uses a flag with an underscore instead of a hyphen. We need to change that to `--user-name`.